draft-ietf-teep-architecture.md

```diff
@@ -114,8 +114,8 @@ sensitive data. An application component running inside a TEE is referred to as
 Trusted Application (TA), while an application running outside any TEE
 is referred to as an Untrusted Application (UA).
 
-The TEE typically uses hardware to enforce protections on the TA and its data, but
-also presents a more limited set of services to applications inside the
+TEEs use hardware enforcement combined with software protection to protect TAs and
```

Comment:
I disagree with the text about hardware enforcement, and prefer the -04 version.  I raised this issue in the TEE Definitions topic at IETF 106.   There are hypervisor based environments that would still need the same TEEP protocol to provision them.  In general the protocol cannot distinguish between hardware based TEEs and software based ones, the only way to distinguish them is at a policy layer that checks for specific keys (e.g., manufacturer keys).  As such, it doesn't make sense for the architecture to constrain things that are purely policy based.

Reply:
I do agree with the "combined with software protection" part though, which I think was your main point in making a change here, so just want to keep the word "typically".

Reply:
The word "typically" does illustrate a majority case, and looks good me.

Reply:
Updated the text based on your feedback.